src/lib/ical/utils.test.ts

```diff
@@ -77,35 +77,35 @@ test('wraps long text into lines broken at column 100 3 times', () => {
   assert(lines.at(-1).length === 1, 'where the last line is a single asterisk');
 });
 
-test('replaces HTML with external links', () => {
-  const actual = replaceHtmlWithExternalLinks(`<a href="https://example.com/">
+test('replaces link targets with "_blank" in HTML string', () => {
+  const actual =
+    replaceHtmlLinkTargets(`<a title="example" target="_self"  href="https://example.com/">Example Link</a>
+  <a title="example" target="_self"href="https://example.com/">Example Link</a>
+  <a target="_self" title="example" href="https://example.com/">Example Link</a>
+  <a title="example" target="_self" target="poggers" target="not poggers"  href="https://example.com/">Example Link</a>
+  <a title="example" target="" href="https://example.com/">Example Link</a>
+  <a href="https://forms.gle/eFpFuiDW6fyRrbUN6">https://forms.gle/eFpFuiDW6fyRrbUN6</a>
```

Comment:
Nit: Use `https://example.com/` for all our sample URLs. To test an example subpage, let's use `https://example.com/example/`.
```suggestion
  <a href="https://example.com/example/">https://example.com/example/</a>
```